### DISCUSSION TITLE
Git Bash syntax error in Oh My Bash 'Sexy' Theme (solved)

### DISCUSSION THREAD

**Discussion by danielferreira2** [ohmybash/oh-my-bash#528] [General]
![screenshot](https://github.com/ohmybash/oh-my-bash/assets/46633039/c4bd7c2e-8221-40ee-9483-a85020ae6f0b)
Hello everyone,

I've recently encountered a syntax error while using the 'Sexy' theme from Oh My Bash on Git Bash for Windows. The error was caused by the way command substitutions were handled in the theme's prompt configuration. Here's a brief overview of the issue and the solution I found:

Issue:
The prompt would throw errors related to command substitution and syntax near unexpected token ) when initializing the Git Bash terminal.

Solution:
I modified the PS1 variable assignment by removing unnecessary escape characters and correcting the command substitution syntax.

``` bash
# Original problematic line:
PS1="$python_venv${MAGENTA}\u ${WHITE}at ${ORANGE}\h ${WHITE}in ${GREEN}\w${WHITE}\$([[ -n \$(_omb_prompt_git branch 2> /dev/null) ]] && echo \" on \")${PURPLE}\$(parse_git_branch)${WHITE}\n\$ ${RESET}"

# Corrected line:
PS1="$python_venv${MAGENTA}\u ${WHITE}at ${ORANGE}\h ${WHITE}in ${GREEN}\w${WHITE}$([[ -n $(_omb_prompt_git branch 2> /dev/null) ]] && echo " on ")${PURPLE}$(parse_git_branch)${WHITE}\n\$ ${RESET}"
```
If this issue has already been resolved, please disregard this message.
Thank you!

**Comment by akinomyoga** [comment]
This is a duplicate of #211, #260, and #512, Read this comment: https://github.com/ohmybash/oh-my-bash/issues/211#issuecomment-1218903376. This is not a syntax error in Oh My Bash, but an old version of Git Bash for Windows has a bug. You can update Git Bash for Windows to the latest version.

```bash
$ git update-git-for-windows
```

**Reply by danielferreira2** [reply]
I have checked my current version and confirmed that I am already on the latest release, 2.43.0.windows.1
![Captura de tela 2024-02-12 001145](https://github.com/ohmybash/oh-my-bash/assets/46633039/8b630b0d-e1a4-45dd-9bbe-8d9a1ad8830b)

**Reply by akinomyoga** [reply]
Does the error happen with the following setting (without loading OMB)?

```bash
# bashrc

PS1='$(echo hello)\n'
```

**Reply by akinomyoga** [reply]
I confirmed the issue in the latest version of Git for Windows. It also seems to happen in the latest version of MSYS2 (Bash 5.2.26). It seems to be reported in https://github.com/msys2/MSYS2-packages/issues/1839, but it's not fixed.
